packages/html2react/src/libraries/__tests__/__snapshots__/component.tests.tsx.snap

```diff
@@ -11,8 +11,12 @@ Array [
     </span>
      for testing purposes.
   </p>,
-  <div
-    className="hello css-12gt2kf"
+  .emotion-0 {
+  width: 180px;
+}
+
+<div
+    className="hello emotion-0"
```

Comment:
This appears now in the snapshots because I've added the https://emotion.sh/docs/@emotion/jest package